src/components/api-reference/api-reference.jsx

```diff
@@ -9,94 +9,82 @@ import Provider from './provider';
 import Submission from './schemas/submission';
 import '../../styles/api-reference/api-reference.css';
 
-function sectionForHash(maybeHash) {
-  const hash = maybeHash || '';
-  if (hash === '#scoring') {
-    return <Scoring/>;
-  } else if (hash === '#provider') {
-    return <Provider/>;
-  } else if (hash.includes('benchmark')) {
-    return <Benchmarks/>;
-  } else if (hash.includes('measurements')) {
-    return <Measurements/>;
-  } else if (hash === '#measurement-sets') {
-    return <MeasurementSets/>;
-  } else if (hash === '#example') {
-    return <ExampleDocs/>;
-  } else {
-    return <Submission/>;
+const ReferenceNavBarItemsMap = {
+  "submission": <Submission />,
+  "measurement-sets": <MeasurementSets />,
+  "measurements": <Measurements />,
+  "benchmarks": <Benchmarks />,
+  "scoring": <Scoring />,
+  "provider-profile-stub": <Provider />
+};
+
+const SamplesNavBarItemsMap = {
+  "example-submission-JSON-&-XML": <ExampleDocs />
+};
+
+function capitalize(string) {
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+function keyToTitle(key) {
+  return key.split("-").map((string) => capitalize(string)).join(" ");
```

Comment:
oo, so "provider-profile-stub" -> "Provider Profile Stub" ? clever, but i wonder if keeping data and logic separate would make it easier to change titles or add i18n later. would a static map of names to titles also work?